Index: trace-cmd-0d25222/Documentation/Makefile
===================================================================
--- trace-cmd-0d25222.orig/Documentation/Makefile
+++ trace-cmd-0d25222/Documentation/Makefile
@@ -46,7 +46,7 @@ endef
 # Most likely a docbook.xsl is already installed on the users system
 # instead of creating a new wheel, lets reuse what's already there.
 #
-MANPAGE_DOCBOOK_XSL =  $(shell locate manpages/docbook.xsl | head -1)
+MANPAGE_DOCBOOK_XSL ?= $(shell locate manpages/docbook.xsl | head -1)
 
 $(obj)/%.xsl: $(src)/%.txt
 	$(Q)mkdir -p $(obj)
